### ISSUE TITLE
On mobile, only the front of cards is shown as an image.

### ISSUE THREAD

**Issue by ThijsMergaert** [ThijsMergaert/realms-card-bot#3]
This is related to https://github.com/discord/discord-api-docs/discussions/3253.

Specifically for Class Ability cards from the Campaign Deck: Ruin of Thandar, this is annoying because the front of the card is the PvE version whereas the back is the PvP version, while the PvP version will generally be of more interest to the bot users.

Proposed workarounds:
- Switch the image order for these ability cards
- Split them into multiple entries

**Comment by tyraziel** [comment]
Both of these solutions/workarounds would require a change to the csv right?  No coding changes needed?

**Comment by ThijsMergaert** [comment]
Not really, we could do either solution in the code or in the data source.  I think I'd prefer to do it in code, since the data source is based on the [public card gallery](https://docs.google.com/spreadsheets/d/1a9MDPhJcEHEkoVvZIT_H4nO4vjSN3VIGlVu4u8ccr8g/edit#gid=0) and making changes to our data source would mean going further out of sync with that, meaning potentially more work for us to repeat if cards are added to that document.

For the reordering work-around, we can just add a check to the end of the `generateCardEmbeds` function.  If the card.Role is 'Character Ability' or 'Character Skill' and the card['Set Name'] is 'Campaign Deck: Ruin of Thandar', change the order of the embeds.

For the splitting into multiple entries, we could do that in the `Gallery.parseGallery` function, but I think it'd be a little more involved.

**Comment by ThijsMergaert** [comment]
Merged the order-reversing workaround as a quick fix, gonna keep this issue open though in case we want to find a better permanent solution for this.
